src/main/java/org/openhab/tools/analysis/checkstyle/DeclarativeServicesDependencyInjectionCheck.java

```diff
@@ -44,6 +44,11 @@
     }
 
     @Override
+    public int[] getAcceptableTokens() {
+        return getDefaultTokens();
```

Comment:
OK for me, but I just wanted to be sure that this will not break if  the default tokens are changed in some of the newer versions.